content/spin/spin-oci.md

```diff
@@ -126,8 +126,7 @@ Lastly, let's run this Spin application.
  <!-- @selectiveCpy -->
 
  ```bash
- # make sure you've built your application with `spin build` prior
- spin up --from-registry ghcr.io/USERNAME/spin-react-fullstack:v1
+ spin oci run --from-registry ghcr.io/USERNAME/spin-react-fullstack:v1
```

Comment:
`spin oci run` has been removed in v0.9.

Reply:
I think this is in an 0.8 tab though - that was my interpretation of lines 124 to here.  Am I misreading?

Reply:
@itowlson, you are reading it correctly.